### PR TITLE
Cultification Additions

### PR DESCRIPTION
**Pull request by Geevies**
* Cultifying an area now cultifies the table and airlocks as well, with some high security exceptions.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/gamemodes/cult/cultify/obj.dm [hunk 1]
```diff
@@ -128,12 +130,17 @@
 	..()
 
 /obj/structure/table/cultify()
-	// Make it a wood-reinforced wooden table.
-	// There are cult materials available, but it'd make the table non-deconstructable with how holotables work.
-	// Could possibly use a new material var for holographic-ness?
-	material = get_material_by_name("wood")
-	reinforced = get_material_by_name("wood")
+	if(material == "cult" || reinforced == "cult")
```
Comment (mikomyazaki) [inline code/game/gamemodes/cult/cultify/obj.dm]: We should really have material defines. Not something you need to do for this PR though.

File: code/game/objects/structures/simple_doors.dm [hunk 2]
```diff
@@ -240,3 +247,7 @@
 
 /obj/structure/simple_door/resin/New(var/newloc,var/material_name)
 	..(newloc, "resin")
+
+/obj/structure/simple_door/cult/New(var/newloc, var/material_name)
+	..(newloc, "cult")
+	color = "#4a3631" // looks better than the standard cult colours
```
Comment (mikomyazaki) [inline code/game/objects/structures/simple_doors.dm]: No one has used this colour before. Maybe add it to `color.dm` and use a define so peeps know what colour it is when reading? (If they're not using a text editor that displays it for you)

File: code/modules/materials/materials.dm [hunk 3]
```diff
@@ -805,7 +807,8 @@ var/list/name_to_material
 	new /obj/effect/decal/cleanable/blood(target)
 
 /material/cult/reinf
-	name = "cult2"
+	name = "cult_reinforced"
+	icon_colour = "#8f3329"
```
Comment (mikomyazaki) [inline code/modules/materials/materials.dm]: Same here.

File: code/game/objects/structures/simple_doors.dm [hunk 4]
```diff
@@ -82,6 +82,11 @@
 /obj/structure/simple_door/attack_hand(mob/user as mob)
 	return TryToSwitchState(user)
 
+/obj/structure/simple_door/attack_generic(mob/user)
+	if(istype(user, /mob/living/simple_animal/construct)) // don't know of any other attack_generic smart enough to open doors
```
Comment (mikomyazaki) [inline code/game/objects/structures/simple_doors.dm]: There's some robotic simple mobs like hivebots, should they be able to open these?

Maybe it should be a var.. or flag... or something. Idk where you'd put it though exactly.

Comment (Geevies) [reply code/game/objects/structures/simple_doors.dm]: nahh, i don't think so. they're not really player controlled either. if someone does add to them later on, they can take a peek.

**Pull request merged**